tools/lint

```diff
@@ -595,7 +595,10 @@ def build_custom_checkers(by_lang):
             "docs/migration-renumbering.md",
             "docs/readme-symlink.md",
             "README.md",
+            "zerver/webhooks/helloworld/doc.md",
+            "zerver/webhooks/solano/doc.md",
             "zerver/webhooks/trello/doc.md",
+            "zerver/webhooks/papertrail/doc.md",
```

Comment:
At some point, we should probably play with the linter settings and/or use full-line markdown links aka the `[foo][bar]` syntax (which escape this linter) to not end up needing these to be excluded.  But this is fine for now.